MyToolsYourToolsClient/src/app/services/auth.service.ts

```diff
@@ -16,7 +16,7 @@ const httpOptions = {
 export class AuthService {
   currentUser: User;
   error: String;
-  baseUrl = 'https://localhost:44341/api/';
+  baseUrl = 'https://localhost:5001/api/';
```

Comment:
Czemu ten url zmieniasz ? u ciebie ten 44341 nie działa ? Bo potem jak merdzujemy to nam znowu nie działa